live-examples/css-examples/fonts/font-optical-sizing.html

```diff
@@ -0,0 +1,25 @@
+<section id="example-choice-list" class="example-choice-list" data-property="font-size">
+
+
+<div class="example-choice">
+<pre><code class="language-css">font-optical-sizing: auto;font-size: 36px;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">font-optical-sizing: none;font-size: 36px;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+</section>
+
+
+<div id="output" class="output hidden">
+    <section id="default-example">
+        <p>Optical sizing example.</p>
```

Comment:
@chrisdavidmills You are missing an id of example-element here. With that, this will work. It seems pretty unclear what the browser support for this is though. It seems to be in Safari and in an insiders version of Edge. Seems not all the things work in Chrome, and it is behind a flag in Firefox although, not sure what that flag is.

Still great to have an example for when it is supported though.